README.md

```diff
@@ -1,1082 +1,876 @@
-<p align="center">
```

Comment:
W tym pliku to się nie da znaleźć miejsca, które chce oznaczyć do poprawki XD
Ale to będzie ta sekcja na dole z autoryzacją.
W ruby jest tak zrobione np. https://github.com/voucherifyio/voucherify-openapi/blob/master/scripts/prepare-open-api-for-sdk/remove-bugged-tags-from-open-api.ts
Proponuje podobnie. Bo ta co teraz jest niewiele mówi.